### DISCUSSION TITLE
Extended creature list?

### DISCUSSION THREAD

**Discussion by doyled-it** [Obsidian-TTRPG-Community/fantasy-statblocks#397] [Q&A] [answered]
There are more creatures than those in the default bestiary, such as those in the [Tome of Beasts](https://www.dndbeyond.com/marketplace/sourcebooks/tome-of-beasts-1) or the [Monster Manual](https://www.dndbeyond.com/marketplace/sourcebooks/monster-manual).

Has anyone created and published extended bestiaries anywhere from any of these source books? I'd like to be able to import them in bulk if someone has already put in the legwork.

**Comment by ThaddaeusWiedemer** [comment, marked as answer]
**Disclaimer: Only import content you own. Support the publishers and content creators.**

There is the [TTRGP convert CLI](https://github.com/ebullient/ttrpg-convert-cli/tree/c1b147b4825684ca009a6d5e84e10ee93de7e65c) you can use for DnD 5e and Pathfinder 2e. It works with Fantasy Statblocks out-of-the-box.

**Reply by doyled-it** [reply]
How do we get JSON versions of creatures? Do they come with the source books?

**Reply by ThaddaeusWiedemer** [reply]
the repo above explains everything, the 5eTools mirror repo has JSONs provided by the community
